### ISSUE TITLE
SIGABRT when running with MXNet 1.3.0.post0

### ISSUE THREAD

**Issue by NRauschmayr** [apache/incubator-mxnet#13448]
Following up from this thread: https://discuss.mxnet.io/t/gluoncv-pikachu-killing-jupyter-on-p3-2x/2309/4

The example code from https://gluon-cv.mxnet.io/build/examples_detection/finetune_detection.html crashes with SIGABRT on a new p2 instance (DLAMI (Ubuntu) 18.0, which comes per default with the MXNet version: 1.3.0.post0). The example works fine with MXNet 1.3.1. 

Find attached the output of strace and gdb. The program crashes because of a memory heap corruption, which triggers abort(). I investigated the issue by setting MALLOC_CHECK_=1, which circumvents the abort(). It seems the issue is caused by the dataloader, but I am not 100% sure.

[strace.txt](https://github.com/apache/incubator-mxnet/files/2626997/strace.txt)
[gdb.txt](https://github.com/apache/incubator-mxnet/files/2627006/gdb.txt)

**Comment by vrakesh** [comment]
@NRauschmayr Thank you for reporting the issue

**Comment by vrakesh** [comment]
@mxnet-label-bot add [Memory]
